### PR TITLE
Nialexsan/page improvements

### PR DESCRIPTION
**Pull request by nialexsan**
- moved contribution guide to CONTRIBUTING.md
- trimmed the home page
- build top nav link

### CODE REVIEW

File: docs/building-on-flow/flow-networks/index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Flow Networks
 sidebar_position: 2
 sidebar_custom_props:
+  description: Mainnet and Testnet overview
```
Comment (gregsantos) [inline docs/building-on-flow/flow-networks/index.md:5]: Develop on Flow Testnet and deploy to Mainnet when production ready.

File: docs/building-on-flow/governance.mdx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Governance
+description: Learn about Flow's governance model and how it empowers the community.
```
Comment (gregsantos) [inline docs/building-on-flow/governance.mdx:3]:
```suggestion
description: Learn about Flow's governance model and how it's empowering our community of users and builders.
```

File: docs/building-on-flow/mobile/index.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: Flow Mobile
 sidebar_position: 20
 sidebar_custom_props:
   icon: 📱
+  description: Discover Flow Mobile, your gateway to a seamless blockchain experience on mobile devices.
```
Comment (gregsantos) [inline docs/building-on-flow/mobile/index.md:6]:
```suggestion
  description: Learn to build applications for iOS and Android by leveraging native SDKs, React Native plugins, and the Flow Core Wallet.
```

File: docs/building-on-flow/run-and-secure/_category_.yml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 label: Running and Securing Flow
 position: 6
 customProps:
-  icon: 🔐
\ No newline at end of file
+  icon: 🔐
+  description: Running and securing your Flow blockchain infrastructure, ensuring robust and secure operation
```
Comment (gregsantos) [inline docs/building-on-flow/run-and-secure/_category_.yml:5]:
```suggestion
  description: Learn about the four Flow node types, and how you can help secure and maintain the Flow blockchain.
```

**Review by gregsantos — CHANGES_REQUESTED** [review]

File: src/ui/design-system/src/lib/Pages/HomePage/index.tsx [hunk 5]
```diff
@@ -24,42 +22,29 @@ export type HomePageProps = SocialLinksSignupProps & {
...
   flips,
   githubUrl,
-  tools,
   upcomingEvents,
 }: HomePageProps): JSX.Element => {
   return (
     <PageBackground gradient="home">
       <LandingHeaderHome
-        description="Discover the developer ecosystem and master the Flow blockchain"
+        description="Start your Flow journey now with the Flow Playground, or explore our comprehensive guides, resources, and references to kickstart your Flow development."
```
Comment (gregsantos) [inline src/ui/design-system/src/lib/Pages/HomePage/index.tsx:40]: @nialexsan  nit: Can you please add a link "Flow Playground" 
I would suggest internally if we have an anchor to it's description or page and externally to the playground itself if necessary.

**Review by gregsantos — APPROVED** [review]
nit but lg2G

File: src/ui/design-system/src/lib/Pages/HomePage/index.tsx (hunk 5 above)
Comment (nialexsan) [reply src/ui/design-system/src/lib/Pages/HomePage/index.tsx]: we don't have a dedicated page for the playground, probably we should
I've put a direct link to https://play.flow.com/

**Pull request merged**